recipes/podofo/all/conanfile.py

```diff
@@ -35,31 +41,17 @@ class PodofoConan(ConanFile):
         "with_tiff": True,
         "with_png": True,
         "with_unistring": True,
+        "with_tools": False,
```

Comment:
Probably we can default to `True`. We usually maximize the components that are built in CCI by default, just in case any consumer needs them.